R/bootstrap.R

```diff
@@ -1422,7 +1422,7 @@ icon <- function(name, class = NULL, lib = "font-awesome") {
   if (!is.null(class))
     iconClass <- paste(iconClass, class)
 
-  iconTag <- tags$i(class = iconClass)
+  iconTag <- tags$i(class = iconClass, role = "presentation", `aria-label` = paste(c(name, "icon"), collapse = " "))
```

Comment:
A bit simpler to do:

```
paste(name, "icon")
```

Unless `name` can have more than one element, but I don't think that's the case here.

Reply:
Thanks! The reason why I did like this was because I wanted to address NULL name so that it can be just "icon" when no name is provided. However, that does not happen, right?

Reply:
Fixed!